builtin/providers/aws/config.go

```diff
@@ -345,35 +349,31 @@ func (c *Config) ValidateCredentials(iamconn *iam.IAM) error {
 
 // ValidateAccountId returns a context-specific error if the configured account
 // id is explicitly forbidden or not authorised; and nil if it is authorised.
-func (c *Config) ValidateAccountId(iamconn *iam.IAM, stsconn *sts.STS, authProviderName string) (string, error) {
+func (c *Config) ValidateAccountId(account_id string) error {
```

Comment:
Again - a nitpick but the convention in this context is also camelCase - i.e. `accountId`.